components/CharacterCard/CharacterCard.js

```diff
@@ -1 +1,28 @@
-export function CharacterCard() {}
+export function CharacterCard({ image, name, status, type, occurrences }) {
+  const newCharacterCard = document.createElement("li");
+  newCharacterCard.className = "card";
+
+  newCharacterCard.innerHTML = `
+          <div class="card__image-container">
+            <img
+              class="card__image"
+              src="${image}"
+              alt="${name}"
+            />
+            <div class="card__image-gradient"></div>
+          </div>
+          <div class="card__content">
+            <h2 class="card__title">${name}</h2>
+            <dl class="card__info">
+              <dt class="card__info-title">Status</dt>
+              <dd class="card__info-description">${status}</dd>
+              <dt class="card__info-title">Type</dt>
+              <dd class="card__info-description">${type}</dd>
+              <dt class="card__info-title">Occurrences</dt>
+              <dd class="card__info-description">${occurrences}</dd>
```

Comment:
As for the occurrences, you should check the API documentation to find out exactly where they are located. :)